app/assets/stylesheets/application.scss

```diff
@@ -48,3 +48,9 @@
 .solicitor_results_list {
   padding-left: 14px;
 }
+
+[data-fn="btn-solicitor-results-close"] {
```

Comment:
according to https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors IE7 does support the [name=value] selector so should be ok